package.yaml

```diff
@@ -44,19 +49,21 @@ tasks:
 jobs:
   package:
     tasks:
+      - docker.Make.build
```

Comment:
I think the plan right now is to use the build artifacts produced by this pipeline rather than what travis uploads. That way we're only using travis to ensure that tests are passing.